ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -309,6 +309,10 @@ host_monitoring_cron:
   minute: "*/5"
   job: ops-runner -f -s 60 -n ozc.send.synthetic.heartbeat.quick /usr/bin/ops-zagg-client -k heartbeat.ping -o 1 --synthetic
 
+- name: Delete stuck projects for bz 1367432
+  minute: "*/5"
```

Comment:
I am assuming this means that we want it to run every 5 minutes. If so, I would suggest we run it every 10 minutes.